sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -1295,6 +1293,172 @@ public async Task AddSettingDefaultAAD()
             }
         }
 
+        [RecordedTest]
+        public async Task CanAddAndGetFeatureFlag()
+        {
+            ConfigurationClient service = GetClient();
+            var testSetting = new FeatureFlagConfigurationSetting(GenerateKeyId("feature 1"), true);
+
+            try
+            {
+                var settingResponse = await service.AddConfigurationSettingAsync(testSetting);
+                var setting = settingResponse.Value;
+
+                Assert.IsInstanceOf<FeatureFlagConfigurationSetting>(setting);
+                Assert.AreEqual(testSetting.Key, setting.Key);
+                Assert.AreEqual(testSetting.Value, setting.Value);
+            }
+            finally
+            {
+                AssertStatus200(await service.DeleteConfigurationSettingAsync(testSetting));
+            }
+        }
+
+        [RecordedTest]
+        public async Task CanAddAndUpdateFeatureFlag()
+        {
+            ConfigurationClient service = GetClient();
+            var testSetting = new FeatureFlagConfigurationSetting(GenerateKeyId("feature 1"), true);
+
+            try
+            {
+                var settingResponse = await service.AddConfigurationSettingAsync(testSetting);
+
+                var flag = (FeatureFlagConfigurationSetting) settingResponse.Value;
+                flag.IsEnabled = false;
```

Comment:
before this, might be worth checking `Assert.True(flag.IsEnabled);` in case someone ever changes the test and passes `false` into the constructor